contributing/running-server-side-experiment.md

```diff
@@ -69,6 +69,19 @@ Example
 }
 ```
 
+In the same pull request you also need to add an experiment constant in the `experiments-const.json` [file](https://github.com/ampproject/amphtml/blob/master/build-system/global-configs/experiments-const.json).
+
+Example
+
+```
+{
+  "MY_EXPERIMENT": false,
+  ...
+}
+```
+
+You start with the `false` value at first. Note that the `experiment-config.json` instructions override the values set in the `experiment-const.json` file.
```

Comment:
Should this be "constants" instead of "instructions"?
```suggestion
You start with the `false` value at first. Note that the `experiment-config.json` constants override the values set in the `experiment-const.json` file.
```